frontend/components/site/UserActionsTable.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { USER_ACTION } from '../../propTypes';
+import { dayAndDate } from '../../util/datetime';
+
+const propTypes = {
+  userActions: PropTypes.arrayOf(USER_ACTION),
+};
+const defaultProps = {
+  userActions: [],
+};
+
+const UserActionsTable = ({ userActions }) =>
+  <table>
+    <caption><b>Action Log</b></caption>
```

Comment:
👍 I think we should open a separate issue to use `<caption>` more broadly (in place of the typical `<h#>` elements we have been using), and style them consistently.

Reply:
Maybe remove the `<b>` for now and rely on a future caption-styling PR, though.

Reply:
I agree!